spark/src/main/scala/eval/directive/RDDOpDirectives.scala

```diff
@@ -1,15 +1,16 @@
 package com.azavea.maml.spark.eval.directive
 
-import cats._
-import cats.data.{NonEmptyList => NEL, _}
-import cats.data.Validated._
-import cats.implicits._
+import com.azavea.maml._
 import com.azavea.maml.ast._
 import com.azavea.maml.eval._
-import com.azavea.maml.eval.directive._
-import com.azavea.maml.spark._
 import com.azavea.maml.spark.eval._
```

Comment:
Ensime's import reformat at work. I like it because it keeps things clean for you without having to reorganize things based on ad-hoc rules.